android-sdk/src/main/java/com/checkout/android_sdk/logging/FramesLoggingEventDataProvider.kt

```diff
@@ -1,18 +1,26 @@
 package com.checkout.android_sdk.logging
+import com.checkout.android_sdk.Utils.CheckoutTheme
 import com.checkout.android_sdk.Utils.Environment
+import com.checkout.android_sdk.Utils.filterNotNullValues
 import com.checkout.eventlogger.domain.model.MonitoringLevel
-import java.util.Locale
+import java.util.*
```

Comment:
It would be nice to avoid wildcard usage in imports when we need just a few classes.